pkg/util/ovs/parse.go

```diff
@@ -361,3 +361,35 @@ func fieldMatches(val, match string, ptype ParseType) bool {
 
 	return false
 }
+
+func ParseExternalIDs(externalIDs string) (map[string]string, error) {
+	ids := make(map[string]string, 1)
+	// Output from "find" and "list" will have braces, but input to "set" won't
+	if externalIDs[0] == '{' && externalIDs[len(externalIDs)-1] == '}' {
+		externalIDs = externalIDs[1:len(externalIDs)-1]
+	}
+	for _, id := range strings.Split(externalIDs, ",") {
+		parsed := strings.Split(strings.TrimSpace(id), "=")
+		if len(parsed) != 2 {
+			return nil, fmt.Errorf("could not parse external-id %q", id)
+		}
+		key := parsed[0]
+		value := parsed[1]
+		if value[0] == '"' {
```

Comment:
ovs.Find() already unquotes the values, is this for list or set op?

Reply:
ovs.Find() unquotes the value if the *whole value* was quoted (meaning, it's a string-value column). But here we're parsing a map-valued column, so ovs.Find won't have removed the internal quotes in the value.